py-polars/tests/unit/io/test_scan.py

```diff
@@ -529,3 +529,14 @@ def test_scan_async_whitespace_in_path(
     assert_frame_equal(pl.scan_parquet(tmp_path / "*").collect(), df)
     assert_frame_equal(pl.scan_parquet(tmp_path / "*.parquet").collect(), df)
     path.unlink()
+
+
```

Comment:
We need to mark this one with `write_disk`

Reply:
ah, yes